src/java.desktop/share/classes/java/awt/image/MultiPixelPackedSampleModel.java

```diff
@@ -448,6 +448,9 @@ public void setSample(int x, int y, int b, int s,
      * not in bounds, or if {@code obj} is not {@code null} or
      * not large enough to hold the pixel data
      * @see #setDataElements(int, int, Object, DataBuffer)
+     * @throws NullPointerException if {@code data} is null.
```

Comment:
what about {@code obj}, guess it will throw NPE too?

Reply:
No, in fact it won't. See the code.